src/main/java/br/com/doit/connector/sentry/SentryConnector.java

```diff
@@ -0,0 +1,55 @@
+package br.com.doit.connector.sentry;
+
+import java.util.HashMap;
+import java.util.Map.Entry;
+
+import org.mule.api.annotations.Config;
+import org.mule.api.annotations.Connector;
+import org.mule.api.annotations.Processor;
+import org.mule.api.annotations.param.Optional;
+
+import io.sentry.SentryClient;
+
+@Connector(name="sentry", friendlyName="Sentry Connector")
+public class SentryConnector {
```

Comment:
@luizalfredo23 we need to write proper documentation if we want to make this project open source. What you think about writing Javadocs for this class? We can take a look at other plugins for inspiration.